tests/MongoDB.Driver.Legacy.Tests/Builders/IndexKeysBuilderTypedTests.cs

```diff
@@ -89,7 +89,14 @@ public void CreateIndex_with_wildcardProjection_should_create_expected_index()
             var indexes = collection.GetIndexes();
             var index = indexes.RawDocuments.Single(i => i["name"].AsString == "custom");
             index["key"]["$**"].AsInt32.Should().Be(1);
-            index["wildcardProjection"].ToBsonDocument().Should().Be(BsonDocument.Parse("{ b : 1, _id : 0 }"));
+            if (CoreTestConfiguration.ServerVersion >= new SemanticVersion(4, 5, 0, ""))
+            {
+                index["wildcardProjection"].Should().Be(BsonDocument.Parse("{ b : true, _id : false }"));
+            }
+            else
+            {
+                index["wildcardProjection"].Should().Be(BsonDocument.Parse("{ b : 1, _id : 0 }"));
+            }
```

Comment:
This change isn't relate to THIS ticket is it?

We could just include it... or it could be a separate ticket.

Reply:
it's cherry-picked from master

Reply:
OK